Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/PropertyEditorNodes.cpp

```diff
@@ -42,8 +42,14 @@ namespace AZ::DocumentPropertyEditor::Nodes
         system->RegisterNodeAttribute<NodeWithVisiblityControl>(NodeWithVisiblityControl::Visibility);
 
         system->RegisterNode<Adapter, NodeWithVisiblityControl>();
+
         system->RegisterNode<Row, NodeWithVisiblityControl>();
 
+        for (const auto& rowAttribute : Row::RowAttributes)
+        {
+            system->RegisterNodeAttribute<Row>(rowAttribute);
```

Comment:
Cool, we might even be able to generalize this to an optional `Attributes` field for node definitions to make registration easier. (Not for this PR)

Reply:
sure, sounds good.